platforms/platform-alexa/src/output/templates/ApiResponseOutput.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+
+export interface ApiResponseOutputOptions extends OutputOptions {
+  apiResponse: Record<string, string | number | boolean>;
+}
+
+@Output()
+export class ApiResponseOutput extends BaseOutput<ApiResponseOutputOptions> {
+  build(): OutputTemplate | OutputTemplate[] {
+    return {
+      listen: this.options.listen,
+      platforms: {
+        alexa: {
+          nativeResponse: {
+            version: '1.0',
```

Comment:
`version` and `sessionAttributes` isn't needed here, will be added automatically

Reply:
let's keep it in there for now